metabrainz/views.py

```diff
@@ -49,3 +49,7 @@ def bad_customers():
 @index_bp.route('/privacy')
 def privacy_policy():
     return render_template('index/privacy.html')
+
+@index_bp.route('/about/customers')
```

Comment:
The old URL was `/about/customers.html`, so it seems you are missing the `.html` here.